.github/workflows/collection-continuous-integration.yml

```diff
@@ -0,0 +1,158 @@
+name: Collection test suite
+
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron: 3 0 * * *  # Run daily at 0:03 UTC
+
+jobs:
+  build-collection-artifact:
+    name: Build collection
```

Comment:
FYI these `name` fields show up in many weird places on the GH UI and most of the time the visual area is limited.
For the best results, invent as short names as possible. That's why I even use single-emoji in some places.

Reply:
I noticed the same, trying to understand which job I'm tracking right now, it's really annoying I can't see the name. Hopefully GH will fix it someday.